models/api-model.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs/promises');
+
+exports.selectApiDocs = () => {
+    return fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
+    .then((fileContents) => {
+        return JSON.parse(fileContents);
+    })
+}
```

Comment:
could just require endpoins.json in, but understand why you've done it this way!